Cargo.toml

```diff
@@ -162,7 +162,7 @@ untrusted = { version = "0.9" }
 [target.'cfg(any(target_arch = "aarch64", target_arch = "arm", target_arch = "x86",target_arch = "x86_64"))'.dependencies]
 spin = { version = "0.9.8", default-features = false, features = ["once"] }
 
-[target.'cfg(all(any(target_os = "android", target_os = "linux"), any(target_arch = "aarch64", target_arch = "arm")))'.dependencies]
+[target.'cfg(all(any(target_os = "android", target_os = "linux", any(target_os = "ios", target_os = "macos", target_os = "tvos")), any(target_arch = "aarch64", target_arch = "arm")))'.dependencies]
```

Comment:
Why repeat `any` inside `any`?

Reply:
Mostly just to keep the apple stuff together. It is kind of frustrating that these are all very similar but need to be spelled out separately everywhere, and this way it is a little easier to (re-)review that it is done correctly whenever we extend it, e.g. to add watchos.